lib/netcdfcxx/patches/config.patch

```diff
@@ -0,0 +1,4826 @@
+diff --git a/config.guess b/config.guess
+index dc84c68..8ef92de 100755
+--- a/config.guess
++++ b/config.guess
```

Comment:
Do not patch generated files!

Reply:
No patches are neccessary by autogen=True